### ISSUE TITLE
Publish ARM compatible linuxstatic binaries + checksums

### ISSUE THREAD

**Issue by mogopz** [pnpm/pnpm#5481]
We build multi-arch base images for our Node applications and would like to include pnpm. This works well for Debian-based images but we're unable to do the same for Alpine images as there are no published ARM compatible `linuxstatic` binaries.

To add on to this it would also be great if checksums could be published alongside the binaries so we're able to verify them.

**Comment by zkochan** [comment]
I believe we can add an artifact for arm64 alpine as vercel/pkg seem to support it: https://github.com/vercel/pkg#targets

**Comment by zkochan** [comment]
Would linuxstatic-arm64 also work? It seems like linuxstatic-x64 works for alpine users.

**Comment by mogggggg** [comment]
> Would linuxstatic-arm64 also work? It seems like linuxstatic-x64 works for alpine users.

Yup, that would be great - can confirm `linuxstatic-x64` works on Alpine

**Comment by zkochan** [comment]
This is implemented in current pnpm, so closing.

pnpm publishes linux-arm64 and linux-arm64-musl artifacts alongside the x64 ones, so Alpine on ARM is covered.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
